README.md

```diff
@@ -179,6 +240,40 @@ To publish changes from the `web-components` subpackage, make sure the version n
 
 [`yarn version`](https://yarnpkg.com/cli/version) is available to use to make changes in the CLI. To change the version of the package you are working on run one of the following commands: `yarn version major`, `yarn version minor`, or `yarn version patch`. For guidance on which command to use please see below.
 
+
+### Releasing
```

Comment:
I made a few additions to these steps because since they were written, we have changed the release PR to increment more than just the `core` version number. And also a couple of clarifying things with what we do after the release is created (PRs for other repos, etc).